assembl/static2/js/app/components/debate/thread/topPostForm.jsx

```diff
@@ -1,193 +1,148 @@
+// FIXME: cancel button
```

Comment:
Did you forget to remove the fixme or to implement it? Or is the implementation of a cancel button out of scope of this story?